R/addLDA.R

```diff
@@ -0,0 +1,105 @@
+#' Latent Dirichlet Allocation
+#'
+#' These functions perform Latent Dirichlet Allocation on data stored in a 
+#'  \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#' 
+#' @param x a \code{\link[TreeSummarizedExperiment:TreeSummarizedExperiment-class]{TreeSummarizedExperiment}}
+#'  object.
+#'   
+#' @param k Integer scalar. A number of latent vectors/topics. 
+#'  (Default: \code{2})
+#' 
+#' @param name Character scalar. The name to be used to store the result in the
+#'  reducedDims of the output. (Default: \code{"LDA"})
+#'  
+#' @param assay.type Character scalar. Specifies which assay to use for LDA 
+#'  ordination. (Default: \code{"counts"})
+#' 
+#' @param ... optional arguments.
```

Comment:
In details section say clearly that topicsmodel::LDA is used

Here say that additional parameters are passed to that function